### PR TITLE
Update LICENSE.txt and 2018 to LICENSE and 2019

### PR DESCRIPTION
**Pull request by teemupiiroinenwirepas**
Signed-off-by: Teemu Piiroinen <teemu.piiroinen@wirepas.com>

### CODE REVIEW

**Comment by teemupiiroinenwirepas** [comment]
> 
> 
> Please make it coherent with the existing license header
> 
> > Copyright Wirepas Ltd 2019 licensed under Apache License, Version 2.0
> > See file LICENSE for full license details.

Done

File: wrappers/python/wirepas_messaging/wnt/__init__.py [hunk 1]
```diff
@@ -5,8 +5,8 @@
     Module to handle WNT authentication and metadata messages
 
     .. Copyright:
-        Copyright 2018 Wirepas Ltd. All Rights Reserved.
-        See file LICENSE.txt for full license details.
+        Copyright 2019 Wirepas Ltd. All Rights Reserved.
```
Comment (PFigs) [inline wrappers/python/wirepas_messaging/wnt/__init__.py]: Should be 

> Copyright Wirepas Ltd 2019 licensed under Apache License, Version 2.0

File: wrappers/python/wirepas_messaging/wnt/ws_api/__init__.py [hunk 2]
```diff
@@ -5,8 +5,8 @@
     WNT websocket metadata messages
 
     .. Copyright:
-        Copyright 2018 Wirepas Ltd. All Rights Reserved.
-        See file LICENSE.txt for full license details.
+        Copyright 2019 Wirepas Ltd. All Rights Reserved.
```
Comment (PFigs) [inline wrappers/python/wirepas_messaging/wnt/ws_api/__init__.py]: Should be

> Copyright Wirepas Ltd 2019 licensed under Apache License, Version 2.0

File: wrappers/python/wirepas_messaging/wnt/ws_api/applicationconfigurationmessages.py [hunk 3]
```diff
@@ -6,8 +6,8 @@
     Application configuration related metadata connection messages
 
     .. Copyright:
-        Copyright 2018 Wirepas Ltd. All Rights Reserved.
-        See file LICENSE.txt for full license details.
+        Copyright 2019 Wirepas Ltd. All Rights Reserved.
```
Comment (PFigs) [inline wrappers/python/wirepas_messaging/wnt/ws_api/applicationconfigurationmessages.py]: same here

File: wrappers/python/wirepas_messaging/wnt/__init__.py (hunk 1 above)
Comment (teemupiiroinenwirepas) [reply wrappers/python/wirepas_messaging/wnt/__init__.py]: Done

File: wrappers/python/wirepas_messaging/wnt/ws_api/__init__.py (hunk 2 above)
Comment (teemupiiroinenwirepas) [reply wrappers/python/wirepas_messaging/wnt/ws_api/__init__.py]: Done

File: wrappers/python/wirepas_messaging/wnt/ws_api/applicationconfigurationmessages.py (hunk 3 above)
Comment (teemupiiroinenwirepas) [reply wrappers/python/wirepas_messaging/wnt/ws_api/applicationconfigurationmessages.py]: Done

**Pull request merged**